configuration/cdap-site.xml

```diff
@@ -1138,117 +1184,350 @@
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>router.connection.backlog</name>
+    <value>20000</value>
+    <description>The connection backlog in the CDAP Router service</description>
+  </property>
+
+  <property>
+    <name>router.connection.idle.timeout.secs</name>
+    <value>15</value>
+    <description>
+      The number of seconds after an HTTP request completes that idle router connections are closed
+    </description>
+  </property>
+
+  <property>
+    <name>router.server.address</name>
+    <value>{{cdap_router_host}}</value>
+    <description>
+      CDAP Router service address to which CDAP UI connects
+    </description>
+  </property>
+
+  <property>
+    <name>router.server.port</name>
+    <value>${router.bind.port}</value>
+    <final>true</final>
     <description>
-    Comma-separated list of CDAP Kafka service brokers; for distributed CDAP, replace with list of FQDN:port brokers
+      CDAP Router service port to which CDAP UI connects
+    </description>
+  </property>
+
+  <property>
+    <name>router.server.worker.threads</name>
+    <value>10</value>
+    <description>The number of worker threads in the CDAP Router service</description>
+  </property>
+
+  <!-- Disable until SSL support
+  <property>
+    <name>router.ssl.bind.port</name>
+    <value>10443</value>
+    <description>
+      CDAP Router service bind port for HTTPS
     </description>
   </property>
 
   <property>
-    <name>log.kafka.topic</name>
-    <value>logs.user-v2</value>
+    <name>router.ssl.server.port</name>
+    <value>${router.ssl.bind.port}</value>
+    <final>true</final>
+  </property>
+  -->
+
+  <!-- Security Configuration -->
+
+  <property>
+    <name>cdap.master.kerberos.keytab</name>
+    <value></value>
+    <description>
+      The full path to the Kerberos keytab file containing the CDAP Master service's
+      credentials
+    </description>
+  </property>
+
+  <property>
+    <name>cdap.master.kerberos.principal</name>
+    <value></value>
+    <description>
+      Example: "CDAP_PRINCIPAL/_HOST@EXAMPLE.COM". The Kerberos primary user
+      that should be used to login to the CDAP Master service. Substitute
+      the Kerberos primary (user) for CDAP_PRINCIPAL, and your domain for
+      EXAMPLE.COM. The string "_HOST" will be substituted with the local
+      hostname.
+    </description>
+  </property>
+
+  <property>
+    <name>kerberos.auth.enabled</name>
+    <value>false</value>
+    <description>
+      Determines if Kerberos authentication is enabled
+    </description>
+    <display-name>Kerberos Enabled</display-name>
+    <value-attributes>
+      <type>boolean</type>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>kerberos.auth.relogin.interval.seconds</name>
+    <value>300</value>
+    <description>
+      Relogin interval for Kerberos keytab
+    </description>
+  </property>
+
```

Comment:
Auth server is CDAP-4110 and isn't implemented. SSL is done in #79 (which will also need a rebase)...